monai/metrics/cumulative_average.py

```diff
@@ -65,6 +65,7 @@ def get_current(self, to_numpy: bool = True) -> NdarrayOrTensor:
         if self.val is None:
             return 0
 
+        val: NdarrayOrTensor
         val = self.val.clone()
```

Comment:
```suggestion
        val: NdarrayOrTensor = self.val.clone()
```
Does combining the annotation with the assignment cause an issue? This should work by deducing the type of the return value of `clone`.

Reply:
Weird indeed, what I suggested should be equivalent but we'll just roll with it for now.